stdlib/experimental/oee/apq_test.flux

```diff
@@ -21,7 +21,7 @@ outData = "
 #group,false,false,false,false,false,false,false,false
 #datatype,string,long,dateTime:RFC3339,double,double,double,double,long
 #default,_result,,,,,,,
-,result,table,_time,availability,oee,performance,quality,runningTime
```

Comment:
Oh, this must be the reason I saw this test was flaky. I tried to look at it because it would randomly fail, but now it makes sense. It was the same diff bug but we never caught it.